src/Kiosk.Api/Services/EctsSubjectService.cs

```diff
@@ -48,16 +62,34 @@ public async Task<bool> AddEctsSubject(EctsSubjectDocument ectsSubjectDocument,
         };
 
         var groupedSubjects = ectsSubjectDocuments.GroupBy(e => new { e.Year, e.Term }).ToList();
-
-        var transformedSubjects = groupedSubjects.Select(x => new SubjectsByYearAndTerm()
+        
+        var transformedSubjects = groupedSubjects.Select(x =>
         {
-            Year = x.Key.Year,
-            Term = x.Key.Term,
-            Subjects = x.Select(subject => _mapper.Map<SubjectResponse>(subject))
+            var subjectByYearAndTerm = new SubjectsByYearAndTerm()
+            {
+                Year = x.Key.Year,
+                Term = x.Key.Term,
+                Subjects = x.Select(subject => _mapper.Map<SubjectResponse>(subject)).OrderBy(x=>x.Subject)
+            };
+
+            var sum = new SubjectResponse()
+            {
+                Subject = "Razem",
```

Comment:
what does 'razem' stand for?

Reply:
to Sum all ects subjects. In next step it will be translated to specific language.
![image](https://github.com/UG-kiosk/API-KIOSK/assets/72080205/243782cf-ae07-4760-a5db-c97800599738)